wurst/systems/modes/Respawn.wurst

```diff
@@ -80,6 +82,25 @@ init
                     timerDialog.display(display)
 
                     delayTimer.doAfter(delay) ->
-                        revive(dyingUnit, owner)
                         timerDialog.display(false)
                         timerDialog.destr()
+
+                        let units = ENUM_GROUP
+                            ..enumUnitsOfForce(owner.getAllies())
+                            .toLinkedList()
+                            ..sortWith((a, b) -> begin
+                                let aType = a.isType(UNIT_TYPE_STRUCTURE) ? 1 : 0
```

Comment:
Could we cast a boolean into an integer here? If so, I think that's more readable.